common/snapshots.py

```diff
@@ -2141,12 +2140,12 @@ def fileInfo(self):
         Load/save "fileinfo.bz2"
 
         Args:
-            d (dict):   dict of: {path: (permission, user, group)}
+            d (FileInfoDict):   FileInfoDict
 
         Returns:
-            dict:       dict of: {path: (permission, user, group)}
+            d:       FileInfoDict
```

Comment:
Docstring format (Google style) for Returns should be `type: description`. So `FileInfoDict: dict of: {path: (permission, user, group)}` in this case